blog_post_collaboration/kedro_ibis.md

```diff
@@ -0,0 +1,341 @@
+│          42 │ Diana      │ S.        │ 2018-02-04  │ 2018-03-12        │                2 │                    27.0 │
+└─────────────┴────────────┴───────────┴─────────────┴───────────────────┴──────────────────┴─────────────────────────┘
+
+```
+
+View the pipeline using Kedro-Viz with `kedro viz run`
+
+<img width="1470" alt="image" src="https://github.com/kedro-org/kedro-devrel/assets/14007150/9fedaf00-ecf6-418f-9c1e-a98844994ba2">
+
+## But wait... there's more!
+
+Making pipeline productionisation easier alone may justify adopting Ibis into a Kedro workflow, but other situations from my past experience also come to mind.
+
+Have you ever found yourself developing data pipelines against an existing data warehouse, even though you know that the data infrastructure team is currently migrating to a different database solution? You expect to rewrite some amount of your code—unless you use Ibis?
+
+Using Ibis can also help you build truly reusable pipelines. I previously led development of a Kedro-based code asset that provided a suite of reusable pipelines for customer analytics. We decided to use PySpark for data engineering, but many of our users didn't need to use PySpark. One major retailer stored all of their data in Oracle, and they ended up with a suboptimal workflow wherein they extracted data into Spark and did all the work there. Other teams looked at our pipelines for inspiration, but ended up rewriting them for their infrastructure—not our intention in delivering prebuilt pipelines. Last but not least, we spent so much time and effort setting up Spark on locked-down Windows machines, so data scientists could play with the pipelines we provided; being able to run the same logic using DuckDB or pandas locally would have been a godsend!
+
+## What's next?
+
+If you're familiar with dbt (or even if you examined the Jaffle Shop project discussed above), you'll notice a key functionality that we didn't implement here: [validations](https://docs.getdbt.com/docs/build/validation). Kedro natively integrates with pytest for unit testing, which plays well for verifying the correctness of transformations developed in Ibis.  Kedro also supports data validation through third-party plugins such as [kedro-pandera](https://kedro-pandera.readthedocs.io/en/latest/), and I've recently started work on extending pandera to support validating Ibis tables; look for a follow-up post covering that soon.
+
+Ibis supports a subset of DDL operations, which means dbt's `incremental` and `materialized view` materialisations currently don't have counterparts yet. Some Ibis backends have explored exposing materialised views. While not explicitly covered above, the `ephemeral` materialisation equates to Kedro's `MemoryDataset`.
+
+Finally, dbt offers enhanced deployment functionality, such as the ability to detect and deploy only modified models; Kedro does not trivially allow detecting changes like this.
```

Comment:
```suggestion
Finally, dbt offers enhanced deployment functionality, such as the ability to detect and deploy only modified models; it's less straightforward to detect changes with Kedro.
```

Reply:
I'm not massively keen on "allow" unless it's a security/permissions kind of thing. How's this change for you?

Reply:
```suggestion
Finally, dbt offers enhanced deployment functionality, like the ability to detect and deploy only modified models; it's less straightforward to detect such changes with Kedro.
```
Seems good! I added a "such" to try to be more clear around what changes we're talking about, but not sure if that's necessary? (Feel free to commit whichever makes more sense to you.)